### ISSUE TITLE
Tile coordinates and lookup tables messed up after "Select best illuminations" execution

### ISSUE THREAD

**Issue by nvladimus** [PreibischLab/BigStitcher#96]
Hi, guys,
I did a processing of multi-tile 2-illumination 2-channel dataset from mesoSPIM microscope [(9 GB dataset link)](https://drive.switch.ch/index.php/s/D2cYOjcridA39pC), and executed "Select best illuminations" function. It is a really cool function (thanks!), and works correctly in general, but after it finished, the tiles were moved very far away from the viewing port, and their lookup tables were reset to 0...65535. I had trouble navigating back to them using the mouse, they were REALLY far:
![BigDataViewer-view](https://user-images.githubusercontent.com/10835134/122776160-206a2580-d2ab-11eb-9389-6cebc011697f.png)
Can anyone please help me? 
The XML file before the transformation is `flipFTF1_000000_ini`, after `flipFTF1_000000`.
Many thanks in advance!

**Comment by hoerldavid** [comment]
Hi Nikita,

About the images being moved away:
Can you try to press ```R``` in the BigStitcher main window? This should re-center the viewport around the currently selected views.

I'll have a look why they are moved in the first place and why we reset the intensities.

Best,
David

**Comment by nvladimus** [comment]
The `R` trick worked, thank you!. I didn't find this shortcut in the BigDataViewer `Help` section, I guess other people just know it :) Thank you very much for looking into this!

**Comment by hoerldavid** [comment]
Yeah, it's not really part of BDV, but something we built right into BigStitcher/MVR - that's why it does not show up in the BDV help I guess.

I had a look and the reason for the resets is that we manually close and re-open BDV, which is necessary to display the updated views (since some will be removed by dropping one of the illuminations): https://github.com/PreibischLab/BigStitcher/commit/770e50722d8bf8785982996c6f0916969e4871f2

I'll look further if we can do the view update in a less disruptive way...

**Comment by nvladimus** [comment]
Ah, I looked into the wrong `Help` section again...
